main/models/answer_model.py

```diff
@@ -0,0 +1,15 @@
+class Answer(object):
+    """ create a model for the answers to the questions. """
+    def __init__(self, user_id, question_id, body):
+        self.user_id = user_id,
+        self.question_id = question_id,
+        self.answer = body
+
+    def answer_per_question(self):
+        self.particular_answer = {
+                "answer": self.answer,
+                "user_id": self.user_id
+                # "question_id": self.question_id
```

Comment:
Why is this code commented? Never push commented code to a repository. Comments are meant to explain why you are doing something not to remove the code you don't need. If you don't need a certain piece of code, Its better you just remove it

Reply:
I have removed the commented code